File: lua/forem-nvim/notify.lua
local M = {}
local function notify(message, level)
  vim.notify(message, vim.log.levels[level], { title = "Forem.nvim" })
end
function M.error(message) return notify(message, "ERROR") end

function M.info(message) return notify(message, "INFO") end

return M

File: src/forem-nvim/notify.ts
type LogLevel = "DEBUG" | "ERROR" | "INFO" | "TRACE" | "WARN" | "OFF";

const notify = (message: string, level: LogLevel): void => {
  vim.notify(message, vim.log.levels[level], { title: "Forem.nvim" });
};

export const error = (message: string): void => notify(message, "ERROR");

export const info = (message: string): void => notify(message, "INFO");
